document/document_test.go

```diff
@@ -434,7 +439,7 @@ func TestNewFromStruct(t *testing.T) {
 			return nil
 		})
 		require.NoError(t, err)
-		require.Equal(t, 27, counter)
+		require.Equal(t, 25, counter)
```

Comment:
Could you explain this change? I am not sure to understand.

Reply:
The previous review, I added some tests  and in this one I delete them. See previous review.